plugins/duck.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: duck
+spec:
+  version: v0.0.1
+  homepage: https://github.com/n3wscott/kubectl-duck
+  shortDescription: List custom resources with ducktype support
+  description: |
+    This plugin allows you to list and get custom resources based on ducktype
+    labels on CRDs.
```

Comment:
```suggestion
    labels on Knative CRDs.
```

Reply:
I think it works on all CRDs (plus maybe core types if OP hardcodes them).

Reply:
it works on all CRDs. At the moment the list of label selectors is limited to ducktypes know by knative, but that will be opened up in future releases.